pages/common/pdftk.md

```diff
@@ -0,0 +1,23 @@
+# pdftk
+
+> PDF toolkit.
+
+- Extract range:
+
+`pdftk {{input.pdf}} cat {{1-3 5 6-10}} output {{output.pdf}}`
+
+- Merge PDF:
+
+`pdftk {{file1.pdf}} {{file2.pdf}} ... cat output {{output.pdf}}`
+
+- Separate PDF into individual pages:
```

Comment:
Are these pages saved as separate pdf files, one per page? And what filename pattern do they use? It would be good for people to be able to predict the outcome of running a command without having to do experiments.